src/js/main.js

```diff
@@ -124,17 +124,17 @@ class Site {
       const day = $('#birthday-day').val();
       const year = $('#birthday-year').val();
       if ( month == null || month == "" || day == null || day == "" || year == null || year == "") {
-        alert('Please fill out all fields.');
+        $('.error-rules').replaceWith("<p>Please fill out all fields</p>");
       } else if (isNaN(month) || isNaN(day) || isNaN(year)) {
-        alert('enter a number!');
+        $('.error-rules').replaceWith("<p>Enter a number!</p>");
       } else {
         const birthday = dayjs(new Date(year, month, day));
         const age = dayjs().diff(birthday, 'years');
         if (age >= 21) {
           Cookies.set('legalAge', true, { expires: 1 }); // Expires in 1 day
           $('body').removeClass('age-check');
         } else {
-        alert('you must be 21 to enter.')
+        $('.error-rules').css("color", "red");
```

Comment:
Better to add a class like `.age-form-error` to the response element when there is a validation error, rather than setting the css in javascript.